tests/cypress/integration/actions_tasks_objects/case_14_appearance_features.js

```diff
@@ -0,0 +1,160 @@
+                cy.contains('Selected opacity').next().within(() => {
+                    cy.get('.ant-slider-step').click('left');
+                    cy.get('[role="slider"]').should('have.attr', 'aria-valuemin').then(($ariaValuemin) => {
+                        ariaValuenow = $ariaValuemin;
+                        cy.get('[role="slider"]').should('have.attr', 'aria-valuenow', ariaValuenow);
+                    });
+                });
+            });
+        });
+        it('Activate the box, the polygon and the cuboid. Boxes are transparent during activated.', () => {
+            for (let i of ['#cvat_canvas_shape_1', '#cvat_canvas_shape_2', '#cvat_canvas_shape_4']) {
+                cy.get(i).trigger('mousemove').should('have.class', 'cvat_canvas_shape_activated').and('have.css', 'fill-opacity', ariaValuenow);
+            }
+        });
+        it('Activate checkbox "show projections".', () => {
+            cy.contains('Show projections').click();
+        });
+        it('Activated the cuboid. Projection lines are visible.', () => {
+            cy.get('#cvat_canvas_shape_4').trigger('mousemove', {force: true})
+            .should('have.attr', 'projections', 'true');
+            cy.get('.cvat_canvas_cuboid_projections').should('be.visible');
+            // Deactivate all objects
+            cy.get('.cvat-canvas-container').click();
+        });
+        it('Activate checkbox "outlined borders" with a red color. The borders are red on the objects.', () => {
+            cy.get('.cvat-objects-appearance-content').within(() => {
+                cy.contains('Outlined borders').click().within(() => {
+                    cy.get('button').click();
+                });
+            });
+            cy.get('.cvat-label-color-picker').within(() => {
+                cy.contains('hex').prev().clear().type(strokeColor);
+                cy.contains('Ok').click();
+            });
+            cy.get('.cvat_canvas_shape').each(object => {
+                cy.get(object).should('have.attr', 'stroke', `#${strokeColor}`);
+            });
+        });
+        it('Set "Color by" to instance. The shapes changed a color.', () => {
+            cy.changeAppearance('Instance');
+            cy.get('.cvat_canvas_shape').each(object => {
+                cy.get(object).should('have.css', 'fill').and('not.equal', fill);
+            });
+        });
+        it('Set "Color by" to group. The shapes are white.', () => {
+            cy.changeAppearance('Group');
+            cy.get('.cvat_canvas_shape').each(object => {
+                cy.get(object).should('have.prop', 'tagName').then($tagName => {
+                    if ($tagName !== 'polyline') {
+                        cy.get(object).should('have.css', 'fill-opacity', '1').and('have.css', 'fill', 'rgb(224, 224, 224)');
+                    } else {
+                        cy.get(object).should('have.css', 'fill-opacity', '0');
```

Comment:
It would be better to check stroke for polylines (with beforehand disabled custom stroke color)

Reply:
Reworked this section of code. Made disabling "Outlined borders" and check css stroke at polyline.

Reply:
Great, thanks!